rust/ruby-rbs/build.rs

```diff
@@ -94,12 +103,67 @@ fn convert_name(name: &str, identifier: CIdentifier) -> String {
         }
     }
 
-    if lowercase {
+    if matches!(identifier, CIdentifier::Type) {
         out.push_str("_t");
     }
     out
 }
 
+fn write_visit_trait(file: &mut File, config: &Config) -> Result<(), Box<dyn std::error::Error>> {
+    writeln!(file, "/// A trait for visiting the AST.")?;
+    writeln!(file, "pub trait Visit {{")?;
+    writeln!(file, "   /// Visits a node.")?;
+    writeln!(file, "   fn visit(&mut self, node: &Node) {{")?;
+    writeln!(file, "       match node {{")?;
+
+    for node in &config.nodes {
+        writeln!(file, "           Node::{}(it) => {{", node.variant_name())?;
+        writeln!(
+            file,
+            "               self.visit_{}_node(it);",
+            convert_name(node.variant_name(), CIdentifier::Method),
+        )?;
+        writeln!(file, "           }}")?;
+    }
+
+    writeln!(file, "       }}")?;
+    writeln!(file, "   }}")?;
+
+    for node in &config.nodes {
+        writeln!(file)?;
+        writeln!(file, "    /// Visits a `{}` node.", node.name)?;
```

Comment:
I think we can probably remove this comment. It basically repeats the same information as the method name.